runtime/README.md

```diff
@@ -281,6 +281,35 @@ Command Code: `0x5349_4756` ("SIGV")
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips\_status | u32      | Indicates if the command is FIPS approved or an error.
 
+### LMS\_SIGNATURE\_VERIFY
+
+Verifies an LMS signature. The hash to be verified is taken from
+Caliptra's SHA384 accelerator peripheral.
+The supported parameter set is limited to: sha256/192, w=4, and h=15 (the same used for the Caliptra image)
+
+Command Code: `0x4C4D_5356` ("LMSV")
+
+*Table: `LMS_SIGNATURE_VERIFY` input arguments*
+
+| **Name**              | **Type** | **Description**
+| --------              | -------- | ---------------
+| chksum                | u32      | Checksum over other input arguments, computed by the caller. Little endian.
+| pub\_key\_tree\_type  | u8[4]    | LMS public key algorithm type. Must equal 12.
+| pub\_key\_ots\_type   | u8[4]    | LM-OTS algorithm type. Must equal 7.
+| pub\_key\_id          | u8[16]   | Private key identifier
```

Comment:
I think the description should say "Public key identifier"

Reply:
This is something I wasn't clear on. That document for LMS (https://datatracker.ietf.org/doc/html/rfc8554#section-5.3)  uses this description for it "The value I is the private key identifier and is the value used for all computations for the same LMS tree." But if that's inaccurate I can absolutely change it.